js/pages/home/routes.js

```diff
@@ -16,7 +16,8 @@ define(
         '/welcome/:token': new Route((token) => {
           require(['welcome'], function () {
             authApi.token(token);
-            document.location = "#/welcome";
+            // application must be reloaded to call bootstrap function again
```

Comment:
It's not clear to me why the bootstrap call is required during OpenID authentication. Could you explain with some more details?

Reply:
the problem is that bootstrap and user/me service is called before route and token is processed so user/me return 401 http code